external_parser/parse_example_binary.h

```diff
@@ -9,6 +9,7 @@
 
 constexpr size_t BINARY_PARSER_VERSION = 1;
 
+constexpr unsigned int MSG_TYPE_FILEMAGIC = 0x42465756; //'VWFB'
```

Comment:
should update `REAME.md` with new file format expectancies

Reply:
OMG, I lost the update on the readme :(

Reply:
done